### DISCUSSION TITLE
Continue the conversation that is ISSUE#68

### DISCUSSION THREAD

**Discussion by i7eo** [Zhengqbbb/cz-git#70] [Q&A] [answered]
The issue is https://github.com/Zhengqbbb/cz-git/issues/68.

I lean command `exec` from [pnpm exec](https://pnpm.io/zh/next/cli/exec). It looks like to be able to execute the bin file？

**Comment by i7eo** [comment]
I test command `pnpm --filter=commitlint-config exec czg` in my project. 

<img width="529" alt="image" src="https://user-images.githubusercontent.com/18899748/195058273-a59e4f84-446e-4c11-8e5d-74f0d79a8cd6.png">


Excute successfully! So I will put the commitlint to my sub package `commitlint-config`👻

**Comment by Zhengqbbb** [comment, marked as answer]
Great ～
`pnpm --filter=foo exec czg` does
`pnpm --filter=foo czg` does not work

Generally I don't like type more word `exec`

**Reply by i7eo** [reply]
I dont't like too. Hope pnpm to delete `exec` and `run` and automatic choose command between bin and scripts.🥶

**Reply by Zhengqbbb** [reply]
I don't think he'll deal with it, as it involves distinguishing conflicts, which would cause a lot of problems if `.bin` and `script` were merged.
They are different concepts.

**Reply by i7eo** [reply]
> I don't think he'll deal with it, as it involves distinguishing conflicts, which would cause a lot of problems if `.bin` and `script` were merged. They are different concepts.

Yep, They are different concepts. I'm just kidding🥶
